cookbook/setup_data/customize_dataset.rst

```diff
@@ -0,0 +1,104 @@
+How to Customize the Dataset
+============================
+
+Akeneo PIM ships with two datasets. They are located in `vendor/akeneo/pim-community-dev/src/Pim/Bundle/InstallerBundle/Resources/fixtures`:
+* `icecat-demo-dev` to be able to take a look and play with the PIM with already preset families, categories, products, etc..
+* `minimal` to start a brand new project
+
+By default, Akeneo PIM ships with `icecat-demo-dev`.
+
+Switch From Icecat to Minimal
+-----------------------------
+
+Edit the file `app/config/parameters.yml` and add the line
+
+.. code-block:: yaml
+
+    installer_data: 'PimInstallerBundle:minimal'
+
+.. note::
+
+   If you use the enterprise version please use `PimEnterpriseInstallerBundle:minimal` instead of `PimInstallerBundle:minimal`.
+
+You can now (re)install your database by running:
+
+.. code-block:: bash
+
+    php app/console pim:installer:db --env=prod
+
+Use Your Own Dataset
+--------------------
+
+The Akeneo PIM allows to prepare a custom data set to use during the installation. The idea is to setup your own catalog structure or your own demo data. The following steps allow you to easily define your own basic entities when you install the PIM.
```

Comment:
not the goal of your PR but minimal should have all the possible files (families, attributes, etc) but only with headers. to help users to start a custom fixtures folder